stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.11.5
+version: 1.12.0
```

Comment:
fyi... there's a merge conflict on these values now because another PR recently versioned the chart to 1.12.0 (so you need to use 1.3.0) and the Traefik software to 1.4.0.

Reply:
@krancour I have done the required changes